addons/mail/static/src/models/message/message.js

```diff
@@ -368,6 +368,18 @@ function factory(dependencies) {
             return `${this.modelName}_${data.id}`;
         }
 
+        /**
+         * @returns {string}
+         */
+        _computeCleanedSubject() {
+            if (this.subject) {
```

Comment:
Do a preventive return if no subject and avoid indenting the rest. Also it should return `clear()` instead of `undefined` in that case.